### DISCUSSION TITLE
Http client connection closure/cancellation forwarding.

### DISCUSSION THREAD

**Discussion by byrnedo** [linkerd/linkerd2#5317] [Q&A] [answered]
We're seeing that, when cancelling a http request proxied via linkerd2-proxy, the downstream service/destination does not see that the client has hung up.
Is there a way to configure linkerd to forward client connection hangups? 
This is http1 btw, not http2 request cancellation.
Running in k8s.

Anyone who can give me some pointers on this? 🙏

**Comment by olix0r** [comment, marked as answer]
@byrnedo We've _just_ fixed the proxy to propagate connection failures back to the client. https://github.com/linkerd/linkerd2/commit/375ffd782f5b952644cc23b9bd5ee208e5a0d8cb#diff-7dbf703cda45892be826e3993fe21c2b0c5af5975ddcf09778c989005912a6fd (via https://github.com/linkerd/linkerd2-proxy/pull/747). This is available in recent edge releases and has been backported to be included in (today's? 🤞) `stable-2.9.1` release.

**Reply by byrnedo** [reply]
Great news @olix0r ! We'll give it a spin and see does it solve the issue 🎉
